### PR TITLE
db: close db when updateOrch prepared statement can not be created

### PR DESCRIPTION
**Pull request by kyriediculous**
**What does this pull request do? Explain your changes. (required)**

correctly close db and return error if the prepared statement for `updateOrch` fails to be bootstrapped. 

**Specific updates (required)**
- handle error for updateOrch prepared statement

**How did you test each of these updates (required)**
ran node in on-chain discovery mode

**Does this pull request close any open issues?**

**Checklist:**
- [ ] README and other documentation updated
- [x] Node runs in OSX and devenv
- [x] All tests in `./test.sh` pass

### CODE REVIEW

File: common/db.go [hunk 1]
```diff
@@ -217,6 +217,11 @@ func InitDB(dbPath string) (*DB, error) {
 		ELSE excluded.deactivationRound END 
 	`)
 	d.updateOrch = stmt
+	if err != nil {
+		glog.Error("unable to prepare updateOrch ", err)
```
Comment (yondonfu) [inline common/db.go]: Capitalize "unable" since all the other error messages used when preparing statements capitalize it?

File: common/db.go [hunk 2]
```diff
@@ -217,6 +217,11 @@ func InitDB(dbPath string) (*DB, error) {
 		ELSE excluded.deactivationRound END 
 	`)
 	d.updateOrch = stmt
+	if err != nil {
```
Comment (yondonfu) [inline common/db.go:219]: Check error immediately after statement preparation to follow the same convention as the rest of the statement preparations?

**Review by yondonfu — CHANGES_REQUESTED** [review]

File: common/db.go (hunk 2 above)
Comment (kyriediculous) [reply common/db.go:219]: fixed

File: common/db.go (hunk 1 above)
Comment (kyriediculous) [reply common/db.go]: fixed

**Review by yondonfu — COMMENTED** [review]
Changes look good! Let's rebase

**Comment by kyriediculous** [comment]
rebased!

**Review by yondonfu — APPROVED** [review]
LGTM 🚢

**Pull request merged**